packages/bpk-component-popover/package-lock.json

```diff
@@ -99,6 +99,11 @@
 			"resolved": "https://registry.npmjs.org/object-assign/-/object-assign-4.1.1.tgz",
 			"integrity": "sha1-IQmtx5ZYh8/AXLvUQsrIv7s2CGM="
 		},
+		"popper.js": {
+			"version": "1.12.9",
+			"resolved": "https://registry.npmjs.org/popper.js/-/popper.js-1.12.9.tgz",
+			"integrity": "sha1-DfvC3/lsRRuzMu3Pz6r1ZtMx1bM="
```

Comment:
Interesting that this is `sha1` like my PR from last week. I feel like we have no actual control over these at this point and they just do whatever sha-type they want..